### ISSUE TITLE
Change "Trolling" to "Flame/rage baiting"

### ISSUE THREAD

**Issue by tzakrajs** [theinternetcafe/code-of-conduct#1]
Does it make sense to change trolling to the more specific flame or rage baiting?

**Comment by redactedaccount** [comment]
Yes. Trolling can be benign and well meaning, akin to harmless practical jokes.

Flame or rage baiting is more appropriate.
